test/test.ts

```diff
@@ -25,6 +25,7 @@ describe('Matrix', () => {
     fromString: input => new WebStreamReader(stringToReadableStream(input))
   }];
 
+  // biome-ignore lint/complexity/noForEach: <explanation>
```

Comment:
Any way we can avoid this ignore?

Reply:
Disabled noForEach rule, it is tiny bit controversial, did not want to change code too much.